aspnetcore/blazor/supported-platforms.md

```diff
@@ -0,0 +1,41 @@
+---
+title: ASP.NET Core Blazor supported platforms
+author: guardrex
+description: Learn about the supported platforms for ASP.NET Core Blazor.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 05/21/2019
+uid: blazor/supported-platforms
+---
+# ASP.NET Core Blazor supported platforms
+
+By [Luke Latham](https://github.com/guardrex)
+
+## Browser requirements
+
+### Blazor client-side
+
+| Browser                          | Version               |
+| -------------------------------- | :-------------------: |
+| Microsoft Edge                   | Current               |
+| Mozilla Firefox                  | Current               |
+| Google Chrome, including Android | Current               |
+| Safari, including iOS            | Current               |
+| Microsoft Internet Explorer      | Not Supported&dagger; |
+
+&dagger;Microsoft Internet Explorer doesn't support [WebAssembly](http://webassembly.org).
+
+For more information on the Blazor client-side hosting model, see <xref:blazor/hosting-models#client-side>.
+
+### Blazor server-side
+
+| Browser                          | Version |
+| -------------------------------- | :-----: |
+| Microsoft Edge                   | Current |
+| Mozilla Firefox                  | Current |
+| Google Chrome, including Android | Current |
+| Safari, including iOS            | Current |
+| Microsoft Internet Explorer      | 11      |
```

Comment:
Let's put a asterix  next to IE11 and add a note that IE11 will require adding additional polyfills.